### ISSUE TITLE
feat[dev]: ✨ implement A11y improvements to sd-interactive

### ISSUE THREAD

**Issue by yoezlem** [solid-design-system/solid#1612]
## Description
This task describes the Storybook implementation for the sd-interactive component as part of the A11y improvements outlined in Epic https://github.com/solid-design-system/solid/issues/1465.

A11y improvements for sd-interactive can be found here: [sd-interactive Notes](https://www.notion.so/matuzo/sd-interactive-111f3f6e1bc08071b9c0c84171dbc365).

-------------------

## Comment
**Problem:** The [icon-only](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/story/components-sd-button-screenshots-sd-button--icon-only) buttons have no accessible name.
**Solution:** Provide an accessible name.

## Note
- The button doesn’t look interactive, especially when displayed without color or for people with a color deficiency.
- Disabled buttons: See [@sd-button](https://www.notion.so/matuzo/sd-button-104f3f6e1bc0800b9f72d2a046316ec3). Everything I wrote there about disabling form elements applies here as well.

## Template Ticket
- [ ] Pick up the template ticket when working on the component story for A11y improvement, as it references the component and will be resolved once the component is made accessible:
#1531 

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by smfonseca** [comment]
@yoezlem @coraliefeil I feel this ticket needs to be addressed first by design since Manuel mentions _it doesn't look interactive, especially when displayed without color or for people with a color deficiency_. In it's current state there is actually no changes required in the code. The fix in the template (subtask) is done in the sd-icon.

**Comment by coraliefeil** [comment]
Icon buttons can take the form of a primary, secondary, tertiary, or cta variant as well as a quaternary variant when used in of sd-interactive. 

We won’t change their visual appearance.

We can give more guidance:

- Only use when the icon’s meaning is unambiguous.
- Consider pairing with a tooltip to describe the icon’s underlying action.

In code we could add browser a tooltip / alt-tag displaying "action name". Hover over your browser icons or icons in other applications ... you will see that they don’t stand out as buttons but show an alt-tag on hover explaining the action of the icon button.
